### DISCUSSION TITLE
How to uninstall

### DISCUSSION THREAD

**Discussion by emrergin** [vkbo/novelWriter#723] [Help] [answered]
Hi, I am not that technically literate, so I don't know whether just deleting the files is enough to uninstall the program.

Thanks.

**Comment by vkbo** [comment, marked as answer]
I need more information to answer that. What operating system did you install on and what install steps did you use?

**Reply by emrergin** [reply]
I am on Windows 10. Used the .bat file to install.

**Reply by vkbo** [reply]
Just delete the folder. The .bat file creates the desktop and start menu icons, which you can just delete as well. A few entries are made in registry also, but those shouldn't cause you any problems.

**Reply by emrergin** [reply]
Thanks for the swift reply, and for this program.

**Reply by AKPhoto** [reply]
Hi sorry to bother but what if you are on ubuntu. I deleted the files to reinstall but now the software won't load.

**Reply by vkbo** [reply]
@AKPhoto check the documentation: https://novelwriter.readthedocs.io/en/stable/setup_linux.html#uninstalling-icons

I'm happy to help, but please respond in the thread you already have on your issue: https://github.com/vkbo/novelWriter/discussions/771
